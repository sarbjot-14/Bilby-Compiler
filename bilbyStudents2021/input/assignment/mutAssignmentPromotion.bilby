main{
	mut b := 5.0; 
	b := 4; 
	print b $n;
	%with 4 being promoted to float 
	
	mut c := 5; 
	c := #a; 
	print c $n;
	
	
}