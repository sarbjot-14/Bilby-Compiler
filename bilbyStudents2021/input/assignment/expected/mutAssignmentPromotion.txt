4.000000
97
